lib/errors.js

```diff
@@ -22,9 +22,11 @@ class CliError extends Error {
         // see https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt
         Error.captureStackTrace(this, this.constructor);
 
-        this.context = options.context || {};
         this.options = options;
         this.help = 'Please refer to https://docs.ghost.org/v1/docs/troubleshooting#section-cli-errors for troubleshooting.'
+
+        // additional error context
+        this.options.context = this.options.context || {};
```

Comment:
When i print `this.context` it's never the context object which is passed in the options. It's always a huge object - not sure where this is coming from.

We also use `this.options.help` [here](https://github.com/TryGhost/Ghost-CLI/blob/1.4.0/lib/errors.js#L59)